OpenOversight/tests/test_utils.py

```diff
@@ -53,13 +53,7 @@ def test_filter_by_badge_no(mockdata):
         assert '12' in str(assignment.star_no)
 
 
-def test_allowed_filenames(app):
-    extension = ['png', 'jpg', 'jpeg', 'mpeg', 'mp4']
-    for e in extension:
-        test_filename = 'test.{}'.format(e)
-	assert OpenOversight.app.utils.allowed_file(test_filename) == True
-
-
-def test_not_allowed_filenames(app):
-    test_filename = 'test.gif'
-    assert OpenOversight.app.utils.allowed_file(test_filename) == False
+def test_hash_file(mockdata):
+    hash_result = OpenOversight.app.utils.hash_file('bacon')
+    expected_hash = '9cca0703342e24806a9f64e08c053dca7f2cd90f10529af8ea872afb0a0c77d4'
+    assert hash_result == expected_hash
```

Comment:
OK so I did almost this but not quite: Added a test for the file storage path in commit 66c2bc9 by monkeypatching the boto3 connection (which actually uncovered a bug which I fixed in 47fdf4b). 

Is the library you mean https://github.com/jubos/fake-s3 which looks like https://github.com/jserver/mock-s3 for python?